.github/workflows/test.yml

```
@@ -35,296 +38,422 @@ concurrency:
           node-version: ${{ matrix.node-version }}
-
       - name: Setup Bun
         uses: oven-sh/setup-bun@v2
         with:
           bun-version: 1.2.5
-
       - name: Install dependencies
         run: bun install
-
+      - name: Ensure test-results directory
+        run: node -e "require('node:fs').mkdirSync('test-results', { recursive: true })"
```

actionlint (1.7.12)

[error] could not parse as YAML: mapping values are not allowed in this context

(syntax-check)